refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -352,61 +341,34 @@ private void fireProgressListenerStop() {
             progressSupport.fireProgressListenerStop(this);
         }
     }
-    
-    private void setWrappers(Transaction commit, UndoableWrapper wrap) {
-        wrap.setActive(true, this);
-        
-        //        if (!(commit instanceof RefactoringCommit))
-        //            return;
-        //        for (FileObject f:((RefactoringCommit) commit).getModifiedFiles()) {
-        //            Document doc = getDocument(f);
-        //            if (doc!=null)
-        //                doc.putProperty(BaseDocument.UndoableEditWrapper.class, wrap);
-        //        }
-    }
-
-    private void unsetWrappers(Transaction commit, UndoableWrapper wrap) {
-        wrap.setActive(false, null);
-
-        //        setWrappers(commit, null);
-    }
-
-    private Document getDocument(FileObject f) {
-        try {
-            DataObject dob = DataObject.find(f);
-            EditorCookie cookie = dob.getLookup().lookup(EditorCookie.class);
-            if (cookie == null)
-                return null;
-            return cookie.getDocument();
-        } catch (DataObjectNotFoundException ex) {
-            return null;
-        }
-    }
 
-    
     private class ElementsCollection extends AbstractCollection<RefactoringElement> {
+
         @Override
         public Iterator<RefactoringElement> iterator() {
             return new Iterator() {
-                //private final Iterator<RefactoringElementImplementation> inner = internalList.iterator();
+
                 private final Iterator<RefactoringElementImplementation> inner2 = SPIAccessor.DEFAULT.getFileChanges(bag).iterator();
                 private int index = 0;
 
                 @Override
                 public void remove() {
                     throw new UnsupportedOperationException();
                 }
-                
+
                 @Override
                 public RefactoringElement next() {
                     if (index < internalList.size()) {
-                        return new RefactoringElement(internalList.get(index++));
+                        RefactoringElementImplementation implementation = internalList.get(index);
```

Comment:
Coding style in effect does not prohibit pre/postfix increments.

Reply:
The IDE complained about it. There is an initialization, a method call, and an increment in one line. Three actions, three lines of code.

Reply:
I may have seen white rabbits. Changed it back, and no complaints at all. Sometimes, mysterious things happen, and I just try to get used to it, alright? You are right. Thank you for the support.